src/docs/templates/collection.hbs

```diff
@@ -0,0 +1,90 @@
+<!DOCTYPE html>
+<html>
+
+<head>
+	{{> html-head title="Collection" }}
+</head>
+
+<body>
+	<div class="container mt-4">
+    <figure class="image-with-text mobile-wrapper row position-relative align-items-center mb-lg-2 mt-5">
+      <div class="col-12 col-lg-8 order-lg-last">
+        <div class="no-gutters__in-container">
+          <picture>
+            <source srcset="https://cdn.shopify.com/s/files/1/0286/1327/9779/files/EUBF_desk_coll_6dcd5236-f00e-438f-9f82-49e405b3240d_791X398.jpg?v=1606285880" media="(min-width: 992px)" />
+            <img src="https://cdn.shopify.com/s/files/1/0286/1327/9779/files/EUBF_mob_coll_706c1f00-877d-44cd-a23e-db861fd1574a_800x.jpg" class="w-100">
+          </picture>
+        </div>
+      </div>
+      <figcaption class="col-12 col-lg-4">
+        <div class="image-with-text__text-content text-left mt-lg-0 p-2 pt-lg-3 pb-lg-3 pr-lg-3 ">
+          <h1>Shop All</h1>
+          <p>Explore award-winning Bali Beauty! Shop Coco & Eve hair, body, tan and gifts. Fast worldwide delivery, unBALIevable results!</p>
+        </div>
+      </figcaption>
+    </figure>
+		<div class="row">
+			<div class="col-3">
+				<ul class="list-unstyled collection-sidebar p-3 d-none d-lg-block">
```

Comment:
utility classes to hide/show should be used on the parent col element, also please use ASIDE for col-3